src/app/EventManagement.h

```diff
@@ -373,6 +373,18 @@ class EventManagement : public DataModel::EventsGenerator
      */
     CHIP_ERROR FabricRemoved(FabricIndex aFabricIndex);
 
+    /**
+     * @brief Iterate all events and remove the events whose associated EventPath is invalid.
```

Comment:
we should probably explain what `Invalid event path` is: it is where a data model provider fails to provide `EventInfo`.

To reduce globals coupling, would it make sense to add the datamodel provide to this method?

Alternatively, this could be a `FilterEvents` and be passed in a filter (function/interface) and let the caller define what we filter out and provide a "events with event info" filter to be used by others.

Reply:
Done